assignments/Action[] actions            # Inherits from Recipe
bool executed
int32 execution_order
